sermant-backend/src/main/java/io/sermant/backend/service/ConfigService.java

```diff
@@ -0,0 +1,356 @@
+            group = NacosUtils.reBuildGroup(request.getGroup());
+        }
+        boolean result = configClient.publishConfig(request.getKey(), group, request.getContent());
+        return new Result<>(ResultCodeType.SUCCESS.getCode(), null, result);
+    }
+
+    /**
+     * Update configuration information
+     *
+     * @param request Configuration information
+     * @return The result of updating configuration information
+     */
+    public Result<Boolean> updateConfig(ConfigInfo request) {
+        Result<Boolean> checkResult = checkConnection(request);
+        if (!checkResult.isSuccess()) {
+            return checkResult;
+        }
+        String group = request.getGroup();
+        if (configClient instanceof NacosClient) {
+            group = NacosUtils.reBuildGroup(request.getGroup());
+        }
+        boolean result = configClient.publishConfig(request.getKey(), group, request.getContent());
+        if (result) {
+            return new Result<>(ResultCodeType.SUCCESS.getCode(), null, true);
+        }
+        return new Result<>(ResultCodeType.MODIFY_FAIL.getCode(), null, false);
+    }
+
+    /**
+     * Delete configuration information
+     *
+     * @param request Configuration information
+     * @return The result of deleting configuration information
+     */
+    public Result<Boolean> deleteConfig(ConfigInfo request) {
+        Result<Boolean> checkResult = checkConnection(request);
+        if (!checkResult.isSuccess()) {
+            return checkResult;
+        }
+        String group = request.getGroup();
+        if (configClient instanceof NacosClient) {
+            group = NacosUtils.reBuildGroup(request.getGroup());
+        }
+        boolean result = configClient.removeConfig(request.getKey(), group);
+        if (result) {
+            return new Result<>(ResultCodeType.SUCCESS.getCode(), null, true);
+        }
+        return new Result<>(ResultCodeType.DELETE_FAIL.getCode(), null, false);
+    }
+
+    /**
+     * Initialize Configuration Center Client
+     */
+    @PostConstruct
+    public void init() {
+        if (!dynamicConfig.isDynamicConfigEnable()) {
+            return;
+        }
+        EXECUTOR_SERVICE.scheduleAtFixedRate(this::reConnection, 0, PERIOD, TimeUnit.SECONDS);
+        String serverAddress = dynamicConfig.getServerAddress();
+        int timeout = dynamicConfig.getTimeout();
+        if (ConfigCenterType.ZOOKEEPER.name().equals(dynamicConfig.getDynamicConfigType())) {
+            watcher = new Watcher() {
+                @Override
+                public void process(WatchedEvent event) {
+                    if (event.getState() == Watcher.Event.KeeperState.Expired) {
+                        configClient = new ZooKeeperClient(serverAddress, timeout, this);
+                    }
+                }
+            };
+            ZooKeeperClient client = new ZooKeeperClient(dynamicConfig.getServerAddress(), dynamicConfig.getTimeout(),
```

Comment:
serverAddress，and timeout，dot need get from dynamicConfig

Reply:
done